github-stats/mysql/README.md

```diff
@@ -0,0 +1,27 @@
+Set-up Google Cloud SQL Proxy
+-----------------------------
+
+Create the database, from the console (recommended) or from command line:
+```
+gcloud sql instances create github-database
+```
+
```

Comment:
Add line about creating service account or choosing existing one? Otherwise not clear which service account user is fetching credentials for.